### PR TITLE
89: old localization logic removed

### PR DESCRIPTION
**Pull request by gitcontributor**
In this PR old localization logic removed from C028 project.
https://github.com/geosolutions-it/MapStore2-C028/issues/89

### CODE REVIEW

**Comment by gitcontributor** [comment]
@allyoucanmap great thanks for your comments and review!
New localization logic introduced in MapSotore2 master branch makes amends to `MetadataExplorer plugin` too (on the core level)  but works in totally another way (it uses query prams when `GetMap` and other requests are introduced).
So I pointed `MetadataExplorer plugin` to submodule source and removed `formatAvailableStyles`

**Comment by allyoucanmap** [comment]
> New localization logic introduced in MapSotore2 master branch makes amends to MetadataExplorer plugin too (on the core level) but works in totally another way (it uses query prams when GetMap and other requests are introduced).
> So I pointed MetadataExplorer plugin to submodule source and removed formatAvailableStyles

@gitcontributor I think we could safely remove also these two components because s/plugins/MetadataExplorer is not imported anymore in plugins configuration
https://github.com/geosolutions-it/MapStore2-C028/blob/master/js/plugins/MetadataExplorer.jsx
https://github.com/geosolutions-it/MapStore2-C028/blob/master/js/components/RecordItem.jsx

File: js/epics/layers.js [hunk 1]
```diff
@@ -73,13 +73,7 @@ const updateMapEpic = (action$, store) =>
...
                             return Rx.Observable.of(caps.error && caps);
                         }
-
-                        // set style by language
-                        const availableStyles = isArray(caps.Style) ? ProjectUtils.formatAvailableStyles(caps.Style) : null;
-                        const currentLocale = head(currentLocaleSelector(store.getState()).split('-'));
-                        const style = ProjectUtils.getLocalizedStyle(layer.style, availableStyles, currentLocale || 'it');
-
-                        return Rx.Observable.of(assign({layer: layer.id, title: ProjectUtils.getKeywordsTranslations(caps), style, availableStyles, bbox: Api.getBBox(caps, true), dimensions: Api.getDimensions(caps)}, (describe && !describe.error) ? {search: describe} : {}));
+                        return Rx.Observable.of(assign({layer: layer.id, title: ProjectUtils.getKeywordsTranslations(caps), bbox: Api.getBBox(caps, true), dimensions: Api.getDimensions(caps)}, (describe && !describe.error) ? {search: describe} : {}));
```
Comment (allyoucanmap) [inline js/epics/layers.js:76]: If we remove the style and availableStyle here we should also remove the style and availableStyles in the updateNode action below in this file
https://github.com/geosolutions-it/MapStore2-C028/blob/b5ef6c50005fd8c094d1a0b5ee00d568da713a83/js/epics/layers.js#L90-L91

**Comment by gitcontributor** [comment]
@allyoucanmap 
Just wondering:
Maybe it is good idea to merge that pr after project update?

**Pull request merged**